package.xml

```diff
@@ -7,11 +7,13 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <depend>geometry_msgs</depend>
   <depend>roscpp</depend>
+  <depend>rostest</depend>
```

Comment:
I think this can be `<test_depend>`

Reply:
fixed aa25d2667f6c14f9c94f2f03b824a2d04051dee5